test/uk/gov/hmrc/taxenrolmentassignmentfrontend/controllers/ReportSuspiciousIDControllerSpec.scala

```diff
@@ -83,6 +100,14 @@ class ReportSuspiciousIDControllerSpec extends TestFixture {
           .returning(createInboundResult(accountDetails))
         mockGetAccountTypeAndRedirectUrlSuccess(randomAccountType)
 
+        val auditEvent = AuditEvent.auditReportSuspiciousPTAccount(
+          accountDetails
+        )(requestWithAccountType(PT_ASSIGNED_TO_OTHER_USER))
+        (mockAuditHandler
+          .audit(_: AuditEvent)(_: HeaderCarrier))
+          .expects(auditEvent, *)
+          .returning(Future.successful((): Unit))
+
```

Comment:
Could you add .once() to eliminate multiple audits not fired.

Reply:
done